src/man/bhyve.5

```diff
@@ -525,7 +559,52 @@ add attr
     set value=/rpool/iso/debian-9.4.0-amd64-netinst.iso
 end
 .Ed
-.sp
+.Pp
+The following example shows how to share a delegated dataset called
+.Pa rpool/datavol
+to a guest using VirtFS.
+This assumes that the mountpoint attribute on
+.Pa rpool/datavol
+is set to
+.Pa /datavol .
+This could have been done, for example, by creating the dataset with:
+.Pp
+.Dl zfs create -o mountpoint=/datavol -o zoned=on rpool/datavol
+.Pp
+Setting the
+.Sy mountpoint
+and
+.Sy zoned
+attributes at the same time prevents the filesystem from ever being mounted in
+the global zone.
+.Bd -literal -offset indent
+add dataset
+    set name=rpool/datavol
+end
+add attr
+    set name=virtfs0
+    set type=string
+    set value=datavol,/datavol
```

Comment:
Is it OK to leave off the "rpool/" from the share name?

Reply:
That isn't a dataset name, it's a mount point where the dataset appears inside the zone. See the example `zfs create` line just above at line 572.

Reply:
Oh I was confused that the first part of the <sharename>,<path> was the dataset (even as I wrote "share name").

Reply:
I’m pretty sure the vm would fail to start with a message in the log that the path doesn’t exist which would make it clear what is wrong.

I guess we could try and be smart about it and try and zfs get the mountpoint if the 2nd value does not start with a / … but that’s a tricky sloop of having way to many ‘smart’ things in boot that should just be clear errors.

Such smart logic probably better suited to be in zadm anyway.

Reply:
makes sense